scripts/zones/The_Boyahda_Tree/mobs/Aquarius.lua

```diff
@@ -11,6 +11,14 @@ entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.SLEEPRES, 100)
```

Comment:
you can set this in mob immunity in mob_pools.sql
Should just be 1 in immunity category